package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@gocrisp/store-locator",
-  "version": "0.5.2",
+  "version": "0.5.4",
```

Comment:
This is what the version already is. There is an issue with the update script, I think related to who can push to `main` - I thought there was a gfe card for that but I don't see it 🤔